src/main/kotlin/uk/gov/justice/digital/hmpps/historicalprisonerapi/model/PrisonerDetail.kt

```diff
@@ -0,0 +1,83 @@
+package uk.gov.justice.digital.hmpps.historicalprisonerapi.model
+
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.Id
+import jakarta.persistence.Lob
+import jakarta.persistence.Table
+import jakarta.validation.constraints.NotNull
+import jakarta.validation.constraints.Size
+import org.hibernate.annotations.Nationalized
+
+@Entity
+@Table(name = "PRISONER_DETAILS", schema = "HPA")
+data class PrisonerDetail(
+  @Id
+  @Size(max = 8)
+  @Column(name = "PK_PRISON_NUMBER", nullable = false, length = 8)
+  val pkPrisonNumber: String,
+
+  @NotNull
+  @Column(name = "PRISONER_DETAILS_ID", nullable = false)
+  val prisonerDetailsId: Int,
+
+  @Size(max = 512)
+  @Nationalized
```

Comment:
What's Nationalized required for? We don't have different character sets do we?

Reply:
thanks, that came through automatically by intellij and hadn't spotted it